### PR TITLE
Bunch of structure changes

### PR DESCRIPTION
**Pull request by paulehoffman**
Added .editorconfig
Moved some defines into main()
Cleaned up all docstrings for function calls
Other little stuff

### CODE REVIEW

File: get_trust_anchor.py [hunk 1]
```diff
@@ -80,15 +73,6 @@
 -----END CERTIFICATE-----
 '''
 
-URL_ROOT_ANCHORS = "https://data.iana.org/root-anchors/root-anchors.xml"
```
Comment (jschlyter) [inline get_trust_anchor.py:83]: Why move constants that may need updating later?

File: get_trust_anchor.py [hunk 2]
```diff
@@ -344,13 +325,18 @@ def get_matching_ksk(KSKRecords, ValidTrustAnchors):
     return MatchedKSKs
 
 
-def export_ksk(ValidKSKs, DSRecordFileName, DNSKEYRecordFileName):
```
Comment (jschlyter) [inline get_trust_anchor.py:347]: The reason for having the filenames as params is that later on we may want to have a command line option to change the filename, please keep them.

File: get_trust_anchor.py [hunk 3]
```diff
@@ -387,6 +373,20 @@ def main():
...
+    URL_ROOT_ANCHORS_SIGNATURE = "https://data.iana.org/root-anchors/root-anchors.p7s"
+    URL_ROOT_ZONE = "https://www.internic.net/domain/root.zone"
+    URL_RESOLVER_API = "https://dns.google.com/resolve?name=.&type=dnskey"
+
+    NowDateTime = datetime.datetime.now()
+    # Date string used for backup file names
+    NowString = "backed-up-at-" + NowDateTime.strftime("%Y-%m-%d-%H-%M-%S") + "-"
+
+    # Make sure there is an "openssl" command in their shell path
+    WhichReturn = subprocess.call("which openssl", shell=True, stdout=subprocess.PIPE)
```
Comment (jschlyter) [inline get_trust_anchor.py:385]: Isn't command not found an exception to the later execution as well?

File: get_trust_anchor.py (hunk 1 above)
Comment (paulehoffman) [inline get_trust_anchor.py:83]: All the constants should be in main() so they are visible to a reader. They are just as editable there as they are at the top of the program. However, it is better to put the ICANN cert at the top of the program to show that it is built in, not retrieved.

File: get_trust_anchor.py (hunk 2 above)
Comment (paulehoffman) [inline get_trust_anchor.py:347]: Yes, correct. If we want the functions to possibly be re-used, the filenames should be in the function call. Please revert this back.

File: get_trust_anchor.py (hunk 3 above)
Comment (paulehoffman) [inline get_trust_anchor.py:385]: It should be tested first before the user gets part-way through running the program.

File: get_trust_anchor.py (hunk 1 above)
Comment (jschlyter) [inline get_trust_anchor.py:83]: Constants defined in main() are not visible to other functions, they need to be global in order to be used. This includes the NowString.

**Comment by jschlyter** [comment]
I've cleaned up, the 'make test' now passes again.

**Pull request merged**